modules/ui/src/app/pages/risk-assessment/risk-assessment.component.ts

```diff
@@ -60,6 +60,24 @@ export class RiskAssessmentComponent implements OnInit, OnDestroy {
     this.store.setFocusOnProfileForm();
   }
 
+  async copyProfileAndOpenForm(profile: Profile) {
+    await this.openForm(this.getCopyOfProfile(profile));
+  }
+
+  getCopyOfProfile(profile: Profile): Profile {
+    const copyOfProfile = { ...profile };
+    copyOfProfile.name = this.getCopiedProfileName(profile.name);
+    return copyOfProfile;
+  }
+
+  private getCopiedProfileName(name: string): string {
+    name = `Copy of ${name}`;
+    if (name.length > 28) {
```

Comment:
I think it will be better to use the variable here. If I got it correct this magic number goes from the validation rule for max length. So, in case this number changes we could update it only in one single place. Suggestion: use single variable for the same/depended logic.